graspologic/inference/latent_position_test.py

```diff
@@ -60,21 +67,32 @@ class LatentPositionTest(BaseInference):
 
     Attributes
```

Comment:
same thing here: functions don't have attributes.

Reply:
so you should have a returns section and describe the three things you are returning